dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/compilation/VisitorPower.java

```diff
@@ -40,6 +41,20 @@ protected VisitorPower(boolean useRC11Scheme, PowerScheme cToPowerScheme) {
         this.cToPowerScheme = cToPowerScheme;
     }
 
+    // =============================================================================================
+    // ========================================= Common ============================================
+    // =============================================================================================
+
+    @Override
+    public List<Event> visitStoreExclusive(StoreExclusive e) {
+        RMWStoreExclusive store = newRMWStoreExclusiveWithMo(e.getAddress(), e.getMemValue(), true, e.getMo());
+
+        return eventSequence(
+                store,
+                newExecutionStatus(e.getResultRegister(), store)
+        );
+    }
```

Comment:
IIRC, power has dependency tracking for failing store conditionals.

Reply:
This was about `RegReaders` of the `regStatus` register depending on the store, right?

Do you have a reference? The only thing I found is in herd7's repo saying that [there is no dependency between the `lwarx` and its matching `stwcx.`](https://github.com/herd/herdtools7/blob/8e7e63cd400e3eae8eac22dfdbd9dae0f243319b/CHANGES.txt#L45).

Reply:
Ok, if the documentation says so, then it is correct I guess :)